conf/config.yaml

```diff
@@ -30,9 +30,6 @@ repos:
  - gdc
  - aws-virginia
 
-portal:
```

Comment:
Are you centralizing these defaults in the context dictionary that Click supports?